example/sysinfo/odp_sysinfo.c

```diff
@@ -407,6 +409,10 @@ int main(void)
 	if (crypto_ret < 0)
 		printf("crypto capability failed\n");
 
+	ipsec_ret = odp_ipsec_capability(&ipsec_capa);
+	if (crypto_ret < 0)
```

Comment:
ipsec_ret, not crypto_ret.

Reply:
Fixed in v2. Also removed a few unnecessary empty lines.